test/python/test_dist_neighbor_loader.py

```diff
@@ -246,13 +248,14 @@ def run_test_as_client(num_servers: int, num_clients: int, client_rank: int, ser
       master_addr='localhost',
       master_port=master_port,
       num_rpc_threads=1,
-      client_group_name='dist-remote-sampling-test-client'
+      client_group_name='dist_remote_sampling_test_client',
+      is_dynamic_world_size=is_dynamic_world_size
     )
 
     print(f'[Client {client_rank}] Creating DistNeighborLoader ...')
     
     options = glt.distributed.RemoteDistSamplingWorkerOptions(
-      server_rank=server_rank,
+      server_rank=server_rank if not server_rank else None, # if auto assign server
```

Comment:
# Automatically assign server_rank (server_rank_list) if server_rank (server_rank_list) is None